components/geometry-merger/dist/aframe-geometry-merger-component.js

```diff
@@ -48,66 +48,90 @@
 	  __webpack_require__(1);
 	}
 
+	AFRAME.utils.setBufferGeometryColor = function () {
```

Comment:
This is a useful utility function to have (the basic example makes use of it). I think it makes sense in AFRAME.utils, rather than being a method on a the geometry-merger component, as it's also useful in cases where geometry-merger is not used (for coloring vertices on any BufferGeometry).

I don't know whether there is any established precedent for extending AFRAME.utils when including a community component, but I don't see a better way of making this available.

(other option might be to extend THREE.BufferGeometryUtils rather than AFRAME.utils???)